ee/modules/110-istio/hooks/hack_iop_reconciling.go

```diff
@@ -0,0 +1,119 @@
+/*
+Copyright 2022 Flant JSC
+Licensed under the Deckhouse Platform Enterprise Edition (EE) license. See https://github.com/deckhouse/deckhouse/blob/main/ee/LICENSE
+*/
+
+package hooks
```

Comment:
Need a description, what hook does and why.

Reply:
added